src/sql/workbench/services/connection/browser/connectionManagementService.ts

```diff
@@ -126,6 +129,30 @@ export class ConnectionManagementService extends Disposable implements IConnecti
 		this._register(this._onDeleteConnectionProfile);
 	}
 
+	/**
+	 * Set the initial value for the connection provider map and listen to the provider change event
+	 */
+	private initializeConnectionProvidersMap() {
+		this.setConnectionProvidersMap();
+		if (this._capabilitiesService) {
+			this._capabilitiesService.onCapabilitiesRegistered(() => {
+				this.setConnectionProvidersMap();
+			});
+		}
+	}
+
+	/**
+	 * Update the map using the values from capabilities service
+	 */
+	private setConnectionProvidersMap() {
```

Comment:
Method doesn't take any arguments. Call it ```refresh``` or ```update```? I think it's reasonable to expect something that starts with ```set``` to accept a new value...